stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2018 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# Copyright 2016-2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    sale_line = fields.Many2one(
+        related='move_id.sale_line_id', readonly=True,
```

Comment:
@ernestotejeda  no need to `readonly=True'  when a field is related

Reply:
@Nikul-Chaudhary I'm sorry, this PR should be closed, it's duplicated, comments should be made at https://github.com/OCA/stock-logistics-reporting/pull/50